src/EnergyPlus/api/datatransfer.h

```diff
@@ -391,6 +391,13 @@ ENERGYPLUSLIB_API Real64 currentTime();
 /// \see apiDataFullyReady
 ENERGYPLUSLIB_API int minutes();
 
+/// \brief Returns the duration of the current zone simulation time step, in fractional hours
+/// \details The zone time step will vary throughout the simulation as needed to maintain convergence while being cautious about program runtime.
+///          This function returns the current value of the time step
+/// \remark The behavior of this function is not well-defined until the `apiDataFullyReady` function returns true.
+/// \see apiDataFullyReady
+ENERGYPLUSLIB_API Real64 zoneTimeStep();
```

Comment:
Not sure of the intent here but TimeStepZone is fixed throughout the simulation (contrary to the comments here). It's TimeStepSys that varies and SysTimeElapsed keeps track of the sub zone time step to the point where SysTimeElapsed = TimeStepZone.

![image](https://user-images.githubusercontent.com/5778156/79931840-f5234680-8419-11ea-99fd-4d09eefde7a2.png)

Reply:
OK, so the next API function, systemTimeStep, returns the system time step so it's just the comments that are wrong above. And maybe a user would need to know the progression of the simulation through the sub zone time step, e.g., ENERGYPLUSLIB_API Real64 systemTimeElapsed();

Reply:
> Not sure of the intent here but TimeStepZone is fixed throughout the simulation (contrary to the comments here). It's TimeStepSys that varies and SysTimeElapsed keeps track of the sub zone time step to the point where SysTimeElapsed = TimeStepZone.
> 
> ![image](https://user-images.githubusercontent.com/5778156/79931840-f5234680-8419-11ea-99fd-4d09eefde7a2.png)

Thanks @rraustad. The intent was to give the user access to the same variables as shown in the EMS App Guide, Table 2.2. I can update the comments to note the fixed timestep.